app/services/midata_service.rb

```diff
@@ -7,7 +7,12 @@ class MidataService
   base_uri ENV.fetch('MIDATA_BASE_URL', 'https://pbs.puzzle.ch')
 
   def initialize(user_email = ENV['MIDATA_USER_EMAIL'], user_token = ENV['MIDATA_USER_TOKEN'], _locale = 'de')
-    @auth_params = { 'X-User-Token': user_token, 'X-User-Email': user_email }
+    # Hitobito deprecated user tokens. But this will not be deployed before BuLa
+    @auth_params = if self.class.base_uri.include?('pbs.puzzle.ch')
+                     { Authorization: "Bearer #{ENV['MIDATA_OAUTH_ACCESS_TOKEN']}" }
```

Comment:
if you ask yourself: sadly the current version of MiData does not support User-Tokens. Service-Tokens do not have enough rights for participants of camps. therefore i made this temporary fix. we might have to use such a token with a very long expiration time as well for the sync.

Reply:
Why did it work until now? Or why didn't we notice?

Reply:
=> pbs.puzzle.ch is already the version where it's not supported whereas https://db.scout.ch still does. as far as i know, they planned to not release before the bula. but we need to make sure!